app/features/analysis/edit_set/OneRMEditSetActions.js

```diff
@@ -75,5 +76,6 @@ const logCloseEditSetAnalytics = (state) => {
         did_change_rpe: didChangeRPE,
         did_change_tags: didChangeTags,
         did_change_reps: didChangeReps,
+        was_removed: wasRemoved,
```

Comment:
Why are we calling it was_removed in our analytics? wouldn't was_error make more sense?

Reply:
- [ ] make it was_error following the v3 analytics doc